src/xenia/base/testing/threading_test.cc

```diff
@@ -154,7 +154,7 @@ TEST_CASE("HighResolutionTimer") {
   // Time the actual sleep duration
   {
     const auto interval = 50ms;
-    std::atomic<uint64_t> counter;
+    std::atomic<uint64_t> counter = 0;
```

Comment:
`counter(0)` should be enough — the initialization doesn't have to be atomic (the assignment would be, I think?)